packages/graphql-cli-graphback-plugin/README.md

```diff
@@ -0,0 +1,36 @@
+## GraphBack
+
+<p align="center">
+  <img width="400" src="https://github.com/aerogear/graphback/raw/master/website/static/img/graphback.png">
+  <br/>
+  Auto generate database structure, <br/>
+  GraphQL Resolvers and Queries from GraphQL types 🚀
+</p>
+
+## Motivation 
+
+GraphQL can be hard and intimidating initially.
+Graphback helps you to kickstart your experience with any existing GraphQL implementation
+by generating backend layer using your data model.
```

Comment:
This description was changed in readme. We changed to to Node.js backend server instead of layer with database changes.